packages/ui/src/bounty/components/tabs/SlashedTab.tsx

```diff
@@ -20,7 +20,7 @@ export const SlashedTab = ({ bounty }: Props) => {
           {slashedWorkers
             .filter((entry) => entry.rejected)
             .map((entry) => (
-              <BountySlashedListItem entrant={entry.worker} inBlock={randomBlock()} link="xd" />
+              <BountySlashedListItem entrant={entry.worker} stake={entry.stake} inBlock={randomBlock()} link="" />
```

Comment:
The PR looks good but is there a way to replace this `randomBlock()` by the actual block of the judgement ?

Reply:
Can do. One more thing about list item, should we remove link since winner list item doesn't have it either?